libs/compact-map/src/Data/Compact/KeyMap.hs

```diff
@@ -24,11 +24,19 @@ import Data.Bits
     (.&.),
     (.|.),
   )
-import Data.Compact.Class
```

Comment:
We can remove the `Data/Compact/Class.hs` module  from the repo now, right?

Reply:
yes